lms/djangoapps/dashboard/tests/test_sysadmin.py

```diff
@@ -162,6 +162,71 @@ def test_staff_access(self):
         response = self.client.get(reverse('gitlogs'))
         self.assertTrue(response.status_code, 200)
 
+    def test_rename_user(self):
+
+        self._setstaff_login()
+        self.client.login(username=self.user.username, password='foo')
+
+        user1 = UserFactory.create(
+            username='test_rename_user_1',
+            email='test_rename_user_1@edx.org',
+            password='foo'
+        )
+        user2 = UserFactory.create(
+            username='test_rename_user_2',
+            email='test_rename_user_2@edx.org',
+            password='foo'
+        )
+
+        new_username = 'targetName'
+        not_found_username = 'notFoundName'
+
+        # ensures that the test database doesn't have a user with username `notFoundName`
+        self.assertEqual(0, len(User.objects.filter(
+            username=not_found_username)))
+
+        # tests response when one field is blank
+        response = self.client.post(reverse('sysadmin'), {
+            'action': 'rename_user',
+            'old_uname': '',
+            'new_uname': new_username,
+        })
+        self.assertIn("Usernames cannot be blank", response.content.decode('utf-8'))
+
+        # tests response when user is not found
+        response = self.client.post(reverse('sysadmin'), {
```

Comment:
See above note about continuation; I think we need to drop the first param down to its own line.